flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/ConfigVariablesManager.java

```diff
@@ -123,6 +129,37 @@ private Property property(String label, String description, Property.ValueType v
         return propertyBuilder.build();
     }
 
+    public JsonElement update(Document document, Path configFile, JsonElement configs) {
+        List<ConfigVariable> configVariables = gson.fromJson(configs, ConfigVariables.class).configVariables();
+        StringBuilder sb = new StringBuilder();
+        for (ConfigVariable configVariable : configVariables) {
+            Map<String, Property> properties = configVariable.properties();
+            String value = properties.get(DEFAULTABLE).toSourceCode();
+            if (value.isEmpty()) {
+                value = "?";
+            }
+            String config = String.format("configurable %s %s = %s;",
+                    properties.get(Property.DATA_TYPE_KEY).toSourceCode(),
+                    properties.get(Property.VARIABLE_KEY).toSourceCode(), value);
+            sb.append(config).append(LS);
```

Comment:
Can we model the `ConfigVariable` as a `FlowNode` since both share the same properties, except for branches, which can be set to null? By doing this, we can reuse existing implementations like the `SourceBuilder` for this case.

Reply:
Added ConfigVariable as a FlowNode